frontend/src/metabase/visualizations/components/FunnelNormal.jsx

```diff
@@ -200,8 +202,16 @@ const GraphSection = ({
   onVisualizationClick,
   className,
 }) => {
+  const rootRef = useRef();
+  useLayoutEffect(() => {
+    // In Safari, the ChartTooltip leaves a trail of artifacts over the Funnel component,
+    // which we fix by redrawing this component when the tooltip moves (i.e. when `hovered` changes).
+    if (isDesktopSafari()) {
+      forceRedraw(rootRef.current);
```

Comment:
when I'm not against such solution completely, if it's the only solution, can't we force redraw in any browser?

Reply:
Are you requesting this change?  I thought it would be best to not redraw unnecessarily, but given the state of browser apps these days I suppose a single wasteful repaint isn’t a big deal.

Reply:
Forcing redraw on every rerender causes layout thrashing. Wdyt about adding css `will-change: transform` to suggest the browser to use a composite layer for the funnel chart only on safari desktop?